lib/internal/errors.js

```diff
@@ -895,6 +894,17 @@ function determineSpecificType(value) {
   return `type ${typeof value} (${inspected})`;
 }
 
+/**
+ * Create a list string in the form like 'A and B' or 'A, B, ..., and Z'.
```

Comment:
I think we should add a comment explaining why we are not using the built-in.
```suggestion
 * Create a list string in the form like 'A and B' or 'A, B, ..., and Z'.
 * We cannot use Intl.ListFormat because it's not available in
 * --without-intl builds.
```